config.go

```diff
@@ -93,6 +93,7 @@ type Settings struct {
 	LoggerConfig            LoggerConfig              `yaml:"logger_config" bson:"logger_config" json:"logger_config" id:"logger_config"`
 	LogPath                 string                    `yaml:"log_path" bson:"log_path" json:"log_path"`
 	Notify                  NotifyConfig              `yaml:"notify" bson:"notify" json:"notify" id:"notify"`
+	OldestAllowedCLIVersion string                    `yaml:"oldest_allowed_cli_version" bson:"oldest_allowed_cli_version" json:"oldest_allowed_cli_version"`
```

Comment:
Can we please add a note to our documentation about this?

Reply:
Can you please also add a note about this to our documentation here?
https://docs.devprod.prod.corp.mongodb.com/evergreen/CLI